app/src/main/java/dracula/vlad/androcalc/MainActivity.java

```diff
@@ -221,65 +223,62 @@ public void getChildViews(ViewGroup viewGroup) {
             int children = viewGroup.getChildCount();
             for (int i = 0; i < children; i++) {
                 View view = viewGroup.getChildAt(i);
+
                 if (view instanceof ViewGroup) {
                     getChildViews((ViewGroup) view);
                 }
+
                 if (view instanceof Button) {
                     TypedValue outValue = new TypedValue();
                     view.getContext().getTheme().resolveAttribute(android.R.attr.selectableItemBackground,
                             outValue, true);
                     view.setBackgroundResource(outValue.resourceId);
```

Comment:
bad idea to setup bg in code. Better practice to create style and apply it in xml to each button